### PR TITLE
TPC Laser calibration - ARGS_ALL_CONFIG fix

### PR DESCRIPTION
**Pull request by rmunzer**

### CODE REVIEW

File: DATA/production/calib/tpc-laser.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ FILEWORKDIR="/home/wiechula/processData/inputFilesTracking/triggeredLaser"
 
 FILEWORKDIR2="/home/epn/odc/files/"
 
-ARGS_ALL_CONFIG="NameConf.mDirGRP=$FILEWORKDIR;NameConf.mDirGeom=$FILEWORKDIR2;NameConf.mDirCollContext=$FILEWORKDIR;NameConf.mDirMatLUT=$FILEWORKDIR;keyval.input_dir=$FILEWORKDIR;keyval.output_dir=/dev/null"
+ARGS_ALL_CONFIG+="NameConf.mDirGRP=$FILEWORKDIR;NameConf.mDirGeom=$FILEWORKDIR2;NameConf.mDirCollContext=$FILEWORKDIR;NameConf.mDirMatLUT=$FILEWORKDIR;keyval.input_dir=$FILEWORKDIR;keyval.output_dir=/dev/null"
```
Comment (martenole) [inline DATA/production/calib/tpc-laser.sh:13]: as above

**Comment by martenole** [comment]
Actually, cannot we drop the current `ARGS_ALL_CONFIG` completely to make the workflows a bit more compact @davidrohr ? At least `mDirGeom` and `mDirGRP` should not be needed anymore since that is taken from CCDB. I don't know where the `input_dir` and `output_dir` are used, but when commenting them out I can still run my workflows without problems

**Comment by davidrohr** [comment]
I think we should clean all this duplicate stuff from the workflows, and the default settings should only come from getCommonArgs.sh.
I would not drop ARGS_ALL_CONFIG alltogether, since it can be used to add configKeyValues globally.
Indeed, we can remove mDirGeam and mDirGRP from getCommonArgs.sh.
For input_dir / output_dir, I am not 100% sure, I would keep that for now.

So we should update `getCommonArgs.sh` and fix these workflows:
```
DATA/production/calib/tpc-laser-filter.sh:ARGS_ALL_CONFIG="NameConf.mDirGRP=$FILEWORKDIR;NameConf.mDirGeom=$FILEWORKDIR2;NameConf.mDirCollContext=$FILEWORKDIR;NameConf.mDirMatLUT=$FILEWORKDIR;keyval.input_dir=$FILEWORKDIR;keyval.output_dir=/dev/null"
DATA/production/calib/tpc-laser.sh:ARGS_ALL_CONFIG="NameConf.mDirGRP=$FILEWORKDIR;NameConf.mDirGeom=$FILEWORKDIR2;NameConf.mDirCollContext=$FILEWORKDIR;NameConf.mDirMatLUT=$FILEWORKDIR;keyval.input_dir=$FILEWORKDIR;keyval.output_dir=/dev/null"
DATA/testing/detectors/FT0/ft0-digits-qc-ctf.sh:ARGS_ALL_CONFIG="NameConf.mDirGRP=$FILEWORKDIR;NameConf.mDirGeom=$FILEWORKDIR;NameConf.mDirCollContext=$FILEWORKDIR;NameConf.mDirMatLUT=$FILEWORKDIR;keyval.input_dir=$FILEWORKDIR;keyval.output_dir=/dev/null"
DATA/testing/detectors/FV0/fv0-digits-qc-ctf.sh:ARGS_ALL_CONFIG="NameConf.mDirGRP=$FILEWORKDIR;NameConf.mDirGeom=$FILEWORKDIR;NameConf.mDirCollContext=$FILEWORKDIR;NameConf.mDirMatLUT=$FILEWORKDIR;keyval.input_dir=$FILEWORKDIR;keyval.output_dir=/dev/null"
DATA/testing/private/afurs/fdd-ft0-fv0-ctf.sh:ARGS_ALL_CONFIG="NameConf.mDirGRP=$FILEWORKDIR;NameConf.mDirGeom=$FILEWORKDIR;NameConf.mDirCollContext=$FILEWORKDIR;NameConf.mDirMatLUT=$FILEWORKDIR;keyval.input_dir=$FILEWORKDIR;keyval.output_dir=/dev/null"
DATA/testing/private/afurs/fdd-ft0-fv0-digits-qc-ctf.sh:ARGS_ALL_CONFIG="NameConf.mDirGRP=$FILEWORKDIR;NameConf.mDirGeom=$FILEWORKDIR;NameConf.mDirCollContext=$FILEWORKDIR;NameConf.mDirMatLUT=$FILEWORKDIR;keyval.input_dir=$FILEWORKDIR;keyval.output_dir=/dev/null"
DATA/testing/private/afurs/ft0-fv0-ctf.sh:ARGS_ALL_CONFIG="NameConf.mDirGRP=$FILEWORKDIR;NameConf.mDirGeom=$FILEWORKDIR;NameConf.mDirCollContext=$FILEWORKDIR;NameConf.mDirMatLUT=$FILEWORKDIR;keyval.input_dir=$FILEWORKDIR;keyval.output_dir=/dev/null"
DATA/testing/private/afurs/ft0-fv0-digits-qc-ctf.sh:ARGS_ALL_CONFIG="NameConf.mDirGRP=$FILEWORKDIR;NameConf.mDirGeom=$FILEWORKDIR;NameConf.mDirCollContext=$FILEWORKDIR;NameConf.mDirMatLUT=$FILEWORKDIR;keyval.input_dir=$FILEWORKDIR;keyval.output_dir=/dev/null"
DATA/testing/private/zampolli/zampolli-workflow.sh:ARGS_ALL_CONFIG="NameConf.mDirGRP=$FILEWORKDIR;NameConf.mDirGeom=$FILEWORKDIR;NameConf.mDirCollContext=$FILEWORKDIR;NameConf.mDirMatLUT=$FILEWORKDIR;keyval.input_dir=$FILEWORKDIR;keyval.output_dir=/dev/null"
```
@martenole : could you take care?

**Comment by martenole** [comment]
Yes, will do

**Pull request merged**